src/main/groovy/com/ullink/Msbuild.groovy

```diff
@@ -240,4 +240,18 @@ class Msbuild extends ConventionTask {
         }
         return cmdParameters
     }
+
+    def getOutputDirs() {
+        def HashMap<String, Object> cmdParameters = getInitProperties()
+        def HashMap<String, String> dirs = ['OutputPath': '.\\build\\bin\\' , 'IntermediateOutputPath' : '.\\build\\obj\\']
```

Comment:
These output folders can have any value in the csproj files, these default/hardcoded paths are unlikely to work except with the most basic projects, don't think we can merge this.

Reply:
we should able to get the property from project properties or task parameters if specified

Reply:
do you have an example @ngyukman maybe we can get that also in csproj file no ?